python/seldon_core/utils.py

```diff
@@ -323,6 +323,7 @@ def construct_response_json(
     client_request_raw: Union[List, Dict],
     client_raw_response: Union[np.ndarray, str, bytes, dict],
     meta: dict = None,
+    metrics: List[Dict] = None,
```

Comment:
Doesnt seem to be used

Reply:
It's used inside the function instead of previously present
```
metrics = client_custom_metrics(user_model)
```
in order to only call `client_custom_metrics` once.